### PR TITLE
Update prod.yaml

### PR DESCRIPTION
**Pull request by Majid-justice**
https://tools.hmcts.net/jira/browse/FACT-940

adding  IDAM_UPDATE_USER_URL: https://idam-api.platform.hmcts.net/api/v1/users/

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by timja — CHANGES_REQUESTED** [review]
Can you adjust the codeowners file please? This shouldn’t require platform ops approval

**Comment by Majid-justice** [comment]
updated codeowners file

**Review by timja — APPROVED** [review]

**Review by NatashaAlker — APPROVED** [review]

**Pull request merged**
